src/sentry/api/endpoints/project_plugin_details.py

```diff
@@ -108,7 +108,7 @@ def put(self, request, project, plugin_id):
             }, status=400)
 
         for key, value in six.iteritems(cleaned):
-            if not value:
+            if value is None:
```

Comment:
i did this so that saving a boolean field as false would work, but not sure if it will break other things?